diff --git a/node_modules/easy-template-x/dist/es/easy-template-x.js b/node_modules/easy-template-x/dist/es/easy-template-x.js
index a1c1723..c15c4ce 100644
--- a/node_modules/easy-template-x/dist/es/easy-template-x.js
+++ b/node_modules/easy-template-x/dist/es/easy-template-x.js
@@ -1540,6 +1540,9 @@ class ImagePlugin extends TemplatePlugin {
                             <a:ext uri="{28A0092B-C50C-407E-A947-70E740481C1C}">
                                 <a14:useLocalDpi xmlns:a14="http://schemas.microsoft.com/office/drawing/2010/main" val="0"/>
                             </a:ext>
+                            <a:ext uri="{96DAC541-7B7A-43D3-8B79-37D633B846F1}">
+                                <asvg:svgBlip xmlns:asvg="http://schemas.microsoft.com/office/drawing/2016/SVG/main" r:embed="${relId}"/>
+                            </a:ext>
                         </a:extLst>
                     </a:blip>
                     <a:srcRect/>
